slack/web/classes/blocks.py

```diff
@@ -78,6 +78,8 @@ def parse(cls, block: Union[dict, "Block"]) -> Optional["Block"]:
                     return InputBlock(**block)
                 elif type == FileBlock.type:
                     return FileBlock(**block)
+                elif type == HeaderBlock.type:
+                    return HeaderBlock(**block)
```

Comment:
This proactive warning by Codecov seems to be a good new feature! To settle this, we can add `Block.parse` in tests as well.

Reply:
The proactive warnings from Codecov are a really nice feature! I've applied the update in your [`tests_block.py` comment](https://github.com/slackapi/python-slackclient/pull/768#discussion_r466060543).